new/components/mgmtworker/scripts/create.sh

```diff
@@ -40,11 +47,23 @@ ctx logger info "Installing Optional Management Worker Modules..."
 [ -z ${SCRIPT_PLUGIN_SOURCE_URL} ] || install_module ${SCRIPT_PLUGIN_SOURCE_URL} ${MGMTWORKER_VIRTUALENV_DIR}
 [ -z ${AGENT_SOURCE_URL} ] || install_module ${AGENT_SOURCE_URL} ${MGMTWORKER_VIRTUALENV_DIR}
 
+# Add certificate and select port, as applicable
+if [[ -n "${RABBITMQ_CERT_PUBLIC}" ]]; then
```

Comment:
This block of code is performed several times, but I'm uncertain that it should be farmed out to a function as it sets two runtime properties, which feels like the sort of side effect a utils function shouldn't have... could go either way on this though.